store/store.ts

```diff
@@ -83,12 +83,26 @@ const rootReducer: Reducer<RootState, AnyAction> = (state, action) => {
   // https://stackoverflow.com/questions/35622588/how-to-reset-the-state-of-a-redux-store/35641992#35641992
   if (isResetStoreAction(action)) {
     const resetState = allSlices.reduce((agg, nextSlice) => {
+      let sliceState = nextSlice.getInitialState()
+
+      // Preserve API key when resetApiKey is false
+      if (nextSlice.name === 'apiKey' && !action.resetApiKey && state?.apiKey) {
+        sliceState = state.apiKey
+      }
+
+      // Preserve configuration settings (useCdnSdk, enableV2View) when clicking "Start Over"
+      // These are user preferences that should persist when creating new businesses in demo
+      if (nextSlice.name === 'kanmonConnect' && !action.resetApiKey) {
```

Comment:
strange to use `resetApiKey` here. maybe we call rename it `completeReset` so it's more generic?